parser-typechecker/src/Unison/FileParser.hs

```diff
@@ -52,9 +52,9 @@ file = do
   --
   -- There's some more complicated logic below to have suffix-based name resolution
   -- make use of _terms_ from the local file.
-  local (\e -> e { names = Names.push (Names.suffixify0 locals) namesStart }) $ do
+  local (\e -> e { names = Names.push locals namesStart }) $ do
     names <- asks names
-    stanzas0 <- local (\e -> e { names = names }) $ sepBy semi stanza
```

Comment:
This `local` call wasn't doing anything! Since it is just setting the `names` it just `ask`d.